app/team.go

```diff
@@ -550,6 +550,30 @@ func GetTeamsUnreadForUser(teamId string, userId string) ([]*model.TeamUnread, *
 	}
 }
 
+func GetTeamUnread(teamId string) (*model.TeamUnread, *model.AppError) {
+	result := <-Srv.Store.Team().GetTeamUnread(teamId);
+	if result.Err != nil {
+		return nil, result.Err
+	}
+
+	data := result.Data.([]*model.ChannelUnread)
```

Comment:
Can you change `data` to `channelUnreads`? Makes it a little more clear what's happening